Code/ForceField/MMFF/TorsionConstraint.h

```diff
@@ -55,6 +55,10 @@ class RDKIT_FORCEFIELD_EXPORT RDKIT_FORCEFIELD_EXPORT TorsionConstraintContrib
   };
 
  private:
+  inline void setParameters(ForceField *owner, unsigned int idx1,
```

Comment:
why `inline`?

Reply:
I added the `setParameters` function to remove some code duplication, but as it is literally only assigning parameters which should not result in a large chunk of code I thought inlining would be good (the compiler will decide for the best anyway). Happy to remove it if you prefer.

Reply:
just doesn't seem like a particularly useful recommendation to the compiler in this case
but I don't feel strongly about it